src/applications/ivc-champva/10-7959C/chapters/healthInsuranceInformation.js

```diff
@@ -37,9 +37,8 @@ we generate the secondary schema). Using this pattern for all primary/secondary
 schemas
 */
 export function applicantHasInsuranceSchema(isPrimary) {
-  const val = isPrimary ? 'Primary' : 'Secondary';
-  const keyname = `applicantHas${val}`;
-  const property = `has${val}`;
+  const keyname = isPrimary ? 'applicantHasPrimary' : 'applicantHasSecondary';
+  const property = isPrimary ? 'hasPrimary' : 'hasSecondary';
```

Comment:
keyname is determined when we call the schema in formconfig, this lets us use the same schema for primary and secondary insurance pages. Adjusted this ternary so the whole keyname will be findable with a search of the codebase